### PR TITLE
2 new graphs and 1 other feature

### PR DESCRIPTION
**Pull request by pncakespoon1**
added enter time distribution graph
added playtime breakdown piechart
added alternative way to split data into session which utilizes session marker, a feature from my tracker

### CODE REVIEW

File: components/user/EntryTypeGraph.jsx [hunk 1]
```diff
@@ -121,6 +149,43 @@ const EntryTypeGraph = ({ data }) => {
...
+                cy="50%"
+                outerRadius={90}
+                fill="#00d0ff"
+              >
+                {biomePieChartData.map((_, idx) => (
+                  <Cell key={`cell-${idx}`} fill={colourList[(idx + 2) % 4]} />
+                ))}
+              </Pie>
+              <Legend layout="horizontal" verticalAlign="bottom" align="right" />
+              <Tooltip />
```
Comment (Specnr) [inline components/user/EntryTypeGraph.jsx]: Tooltip needs to be formatted, see how its done in the bar graphs above 

image

**Review by Specnr — CHANGES_REQUESTED** [review]
is this correct? Im assuming x axis is time in seconds and y axis is enter count, but i dont think boeynn has 2598 enters at 2m.

image

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (Specnr) [inline .vscode/settings.json]: also we dont need this file

File: components/user/EntryTypeGraph.jsx [hunk 3]
```diff
@@ -40,6 +40,34 @@ const EntryTypeGraph = ({ data }) => {
...
+  playtimePieChartData.push({
+    name: "Nether",
+    percOfTotal: data.nt
+  })
+  playtimePieChartData.push({
+    name: "Wall",
+    percOfTotal: data.wt
+  })
+
+  const netherDistData = []
```
Comment (Specnr) [inline components/user/EntryTypeGraph.jsx]: This should be done on the backend, its rather intensive

File: components/user/EntryTypeGraph.jsx (hunk 1 above)
Comment (pncakespoon1) [reply components/user/EntryTypeGraph.jsx]: tooltip is now formatted, the first graph is cummulative

File: components/user/EntryTypeGraph.jsx (hunk 3 above)
Comment (pncakespoon1) [reply components/user/EntryTypeGraph.jsx]: will have you change that or I will change later

**Comment by Specnr** [comment]
old PR branch, new PR will include these changes

**Pull request closed without merging**